src/redux/EditorSlice.js

```diff
@@ -264,16 +265,18 @@ export const EditorSlice = createSlice({
     updateProjectComponent: (state, action) => {
       const extension = action.payload.extension;
       const fileName = action.payload.name;
-      const code = action.payload.code;
+      const content = action.payload.content;
+      const cascadeUpdate = action.payload.cascadeUpdate;
```

Comment:
more a style thing I guess, but for brevity you could use something like `const { extension, name: fileName, content, cascadeUpdate } = action.payload;`

Reply:
Sounds good, thanks 👍